dev-docs/RFCs/v6.1/json-layers-rfc.md

```diff
@@ -0,0 +1,105 @@
+# RFC: JSON Layers
+
+* **Authors**: Ib Green
+* **Date**: July 2018
+* **Status**: Draft Proposal
+
+
+## Summary
+
+This RFC proposes adding adaptor classes that accept JSON descriptions of views and layers.
```

Comment:
What kind of JSON schema would you expect? Something like vega schema?
https://vega.github.io/schema/vega/v2.6.5.json

Reply:
> What kind of JSON schema would you expect? 

Great question! 

In my mind this relates to the "ONE API" principle mentioned at the end of the RFC. Even though deck.gl comes in many incarnations (pure JS, scripting, React, JSON) we want all these APIs to be the "same" or at least similar with very trivial mappings whenever possible. 

That said, we should compare with existing schemas and see how they have solved tricky points, like interactivity etc.

We could of course in addition also support any separate arbitrary schema type(s) we want, but I would really prefer the "core schema" to be very close to the "core API".